Common/OpenCL/ITKimprovements/itkGPUImage.h

```diff
@@ -221,6 +221,19 @@ class ITKOpenCL_EXPORT GPUImage : public Image< TPixel, VImageDimension >
   itkGetConstReferenceMacro( IndexToPhysicalPoint, DirectionType );
   itkGetConstReferenceMacro( PhysicalPointToIndex, DirectionType );
 
+  /** GPUImage counterpart*/
+  template <typename UPixelType, unsigned int NUImageDimension = VImageDimension>
+  struct Rebind
+    {
+      using Type = itk::GPUImage<UPixelType, NUImageDimension>;
+    };
+
+
+  template <typename UPixelType, unsigned int NUImageDimension = VImageDimension>
+    using RebindImageType = itk::GPUImage<UPixelType, NUImageDimension>;
+
+
+
```

Comment:
I do not understand this part. @dpshamonin could you verify this? @N-Dekker could you check with Denis?